lib/cypress_on_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module CypressOnRails
-  VERSION = '1.13.1'.freeze
+  VERSION = '1.14.1'.freeze
```

Comment:
I like to keep versioning in a separate commit.

```suggestion
  VERSION = '1.13.1'.freeze
```